snap/gui/oudedetai.verbum.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Verbum Bible
+GenericName=FaithLife Wine App Installer
+Comment=Manages FaithLife Bible Software via Wine (snap)
+Exec=oudedetai.verbum
```

Comment:
Where is this stored?

Assuming this is this the verbium app in snap.yml?

Reply:
Yes, `snapd` doesn't recoginize new desktop files placed inside of the snap app's user's `.local/share/applications` folder (e.g. `~/snap/oudedetai/current/.local/share/applications`), so the desktop files need to be installed when the snap package is installed.

The `snap/snapcraft.yaml` then included the normal `oudedetai` command, plus two subcommands: `oudedetai.logos` and `oudedetai.verbum` that these desktop files will run. Those subcommands then run `run.sh` found in `snap/bin`, which runs the installed Faithlife app. If the app isn't installed, it first runs the `oudedetai --install-app` command.